en/docs/try-out/payment-initiation-service-flow.md

```diff
@@ -613,3 +613,352 @@ In this section, you will be generating an access token using the authorization
+   },
+   "creditorName": "Merchant123",
+   "transactionStatus": "ACCP",
+   "endDate": "2023-09-10",
+   "dayOfExecution": "01",
+   "creditorAccount": {
+       "iban": "DE98765432109876543210"
+   },
+   "instructedAmount": {
+       "amount": "123.50",
+       "currency": "EUR"
+   },
+   "remittanceInformationUnstructured": "Ref Number Abonnement",
+   "startDate": "2023-09-05",
+   "frequency": "Monthly"
+}
+```
+
+!!! note
+    - This response is constructed by the core banking back end.
+    - The `transactionStatus` property shows the status of the real payment at the time the request arrives at the bank backend.
+    - The response to the request must comply with the response specified in the specification.
+
+**GET /{payment-service}/{payment-product}/{paymentId}/status**
+
+This request is applicable for instant payments, bulk payments, and periodic payments.
+
+Given below is a sample request for a periodic payment:
+
+``` tab="Request"
+curl -X GET 'https://localhost:8243/xs2a/v1/periodic-payments/sepa-credit-transfers/31169aed-3a46-4404-9035-99de2e776a60/status' \
+-H 'PSU-IP-Address: 127.0.1.1' \
+-H 'PSU-ID-Type: email' \
+-H 'X-Request-ID: d029c37a-d464-4304-b175-7c27fd7f5728' \
+-H 'Date: Mon, 31 Oct 2022 12:12:08 GMT' \
+-H 'PSU-ID: admin@wso2.com' \
+-H 'Signature: keyId="SN=de730ec5733ead1b,CA=1.2.840.113549.1.9.1=#16116d616c7368616e694077736f322e636f6d,CN=OpenBanking Pre-Production Issuing CA,OU=OpenBanking,O=WSO2 (UK) LIMITED,L=COL,ST=WP,C=LK",algorithm="rsa-sha256", headers="digest x-request-id date psu-id",signature=AEtotwhYvGkGQEZYD0a/rvBwW6JCqJ2hBFha8lQdcSI/zruoxF2cYkeRNSir3pwTYF9QmYwp2aPJZ8LoAozSHNQUs2Q9BmNHVET/g8HFV+7iBzfEqK6Mtyzu0u9u7qhBaLdu3vq1gSyMuQHPFoKgWgTGGVFiL0cTICp1Zka5niCxa6GGGwpwvER9DrApSk3nvwm/+vwHwmEEeP3LoQubvKDyDF6Vj438VLKpMGBl1kWv/WwkLhIUSW6OSVvxX+xWhgYBS0qxEW8FroSJIul1ncbhst8qAgFCmoV4fSon1o9qaEHQwnnIec/qpldbtEOmEZNGVCmnfu2OY1XyVw3Vxw==' \
+-H 'Digest: SHA-256=RBNvo1WzZ4oRRq0W9+hknpT7T8If536DEMBg9hyq/4o=' \
+-H 'TPP-Signature-Certificate: <SIGNATURE_CERTIFICATE>' \
+-H 'Authorization: Bearer <APPLICATION_ACCESS_TOKEN>'
+```
+
+``` tab="Response"
+200 OK
+Header: X-Request-ID: d029c37a-d464-4304-b175-7c27fd7f5728
+{
+   "transactionStatus": "ACCP"
+}
+```
+
+!!! note
+    - This response is constructed by the core banking back end.
+    - The `transactionStatus` property shows the status of the real payment at the time the request arrives at the bank backend.
+    - The response to the request must comply with the response specified in the specification.
+
+### Payment cancellation requests
+
+!!! info
+    This is only available as a WSO2 Update from **WSO2 Open Banking Identity Server Berlin Toolkit Level 1.0.0.5** and
+    **WSO2 Open Banking API Manager Berlin Toolkit Level 1.0.0.5** onwards. For more information on updating, see
+    [Getting WSO2 Updates](../install-and-setup/setting-up-servers.md#getting-wso2-updates).
+
+!!! tip "Before you begin:"
+    1. Update the following files:
```

Comment:
Shall we tell to update the following files with the files in the latest update level (1.0.0.5 or later)?

Reply:
Fixed in 4d634c22b28377dd06f03734b943bf636eac4115.